examples/standalone/benchmarks/run_on_daint.sh

```diff
@@ -83,19 +88,39 @@ else
   githash="notarepo"
 fi
 
-# Adapt batch script:
-cp buildenv/submit.daint.slurm .
-sed -i s/\<NAME\>/standalone/g submit.daint.slurm
-sed -i s/\<NTASKS\>/$ranks/g submit.daint.slurm
-sed -i s/\<NTASKSPERNODE\>/1/g submit.daint.slurm
-sed -i s/\<CPUSPERTASK\>/$nthreads/g submit.daint.slurm
-sed -i s/--output=\<OUTFILE\>/--hint=nomultithread/g submit.daint.slurm
-sed -i s/00:45:00/03:30:00/g submit.daint.slurm
-sed -i s/cscsci/normal/g submit.daint.slurm
-sed -i s/\<G2G\>//g submit.daint.slurm
-sed -i "s#<CMD>#export PYTHONPATH=/project/s1053/install/serialbox2_master/gnu/python:\$PYTHONPATH\nsrun vcm_1.0/bin/python examples/standalone/runfile/dynamics.py test_data/ $timesteps $backend $githash#g" submit.daint.slurm
+
+
+echo "submitting script to do compilation"
+# Adapt batch script to compile the code:
+sed -i s/\<NAME\>/standalone/g compile.daint.slurm
+sed -i s/\<NTASKS\>/$ranks/g compile.daint.slurm
+sed -i s/\<NTASKSPERNODE\>/$ranks/g compile.daint.slurm
+sed -i s/\<CPUSPERTASK\>/1/g compile.daint.slurm
+sed -i s/--output=\<OUTFILE\>/--hint=nomultithread/g compile.daint.slurm
+sed -i s/00:45:00/03:30:00/g compile.daint.slurm
+sed -i s/\<G2G\>/export\ CRAY_CUDA_MPS=1/g compile.daint.slurm
+sed -i "s#<CMD>#export PYTHONPATH=/project/s1053/install/serialbox2_master/gnu/python:\$PYTHONPATH\nsrun python examples/standalone/runfile/dynamics.py test_data/ 1 $backend $githash#g" compile.daint.slurm
+
+# execute on a gpu node
+sbatch -W -C gpu compile.daint.slurm
+wait
+echo "compilation step finished"
+rm *.json
+
+echo "submitting script to do performance run"
+# Adapt batch script to run the code:
+sed -i s/\<NAME\>/standalone/g run.daint.slurm
+sed -i s/\<NTASKS\>/$ranks/g run.daint.slurm
+sed -i s/\<NTASKSPERNODE\>/1/g run.daint.slurm
+sed -i s/\<CPUSPERTASK\>/12/g run.daint.slurm
+sed -i s/--output=\<OUTFILE\>/--hint=nomultithread/g run.daint.slurm
+sed -i s/00:45:00/00:30:00/g run.daint.slurm
+sed -i s/cscsci/debug/g run.daint.slurm
```

Comment:
Does the `debug` queue work with multiple nodes?

Reply:
it does work. But I just compared with our old data where a timestep is still close to 45' with the c128 data, i switched it to normal.